examples/python/src/utils/frontify_queries/create_asset/create_asset.py

```diff
@@ -0,0 +1,82 @@
+import os
+
+from src.services.FrontifyClient import FrontifyClient
+from src.utils.frontify_queries.create_asset.create_asset_executor import (
+    create_asset_executor,
+)
+
+
+def create_asset(
+    client: FrontifyClient,
+    projectid: str,
+    fileid: str,
+    title: str = None,
+    filename: str = None,
+    directory: str = None, # directory path in the format "folder1/folder2"
+    description: str = "",
+    author: str = "",
+    copyright: str = None,
```

Comment:
```suggestion
    copyright: str = 'UNKNOWN',
```
As per our docs, the default value is 'UNKNOWN'
This also helps with the Copyright object construction a few lines down from here.